data/innspark.mdx

```diff
@@ -0,0 +1,23 @@
+---
+name: Innspark Solutions
+description: Innspark is known as a passionate Deeptech company among its customers for providing highest quality products for solving the most difficult challenges facing the customers. This is possible by the Sincere, Passionate and hardworking Nerds, Engineers & Researchers. Solving the problems is part of their DNA. Innspark's products and Services have won several coveted awards at the national level and has received several compliments from its customers for solving some of the toughest problems. The experience of our passionate in-house researchers who have developed and implemented technology solutions to a wide range of customers with highly complex environments including government organizations, banks, PSUs, and hospitals enables us to provide services for organizations of virtually any size.
+founded: 19 Feb 2019
+based: Both
+domain: Computer and Network Security
+logo: innspark.svg
+location: Kollam
+address: CP/XII/482,Clappana PO
+Karunagapally,Kollam
```

Comment:
@SuvarneshKM There is a small parsing error that occur here

Reply:
ok